backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -321,6 +322,23 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
         context.cAdapterGenerator.generateBindings(codegen)
     }
 
+    private fun withStaticInitializer(module: ModuleDescriptor, f: () -> Unit) {
```

Comment:
The name is kinda confusing. Maybe something like `runAndProcessInitializers`?

Reply:
Thanks, I was struggling to come up with a sane name for this.